.travis.yml

```diff
@@ -11,3 +11,14 @@ branches:
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
+deploy:
+  provider: s3
+  access_key_id: AKIAIRWAFHZDDWJ37X5A
```

Comment:
Consider a comment here pointing the reader to the README if they want instructions on updating this field.